ocaml/idl/datamodel_pool.ml

```diff
@@ -803,6 +803,14 @@ open Datamodel_types
       ~allowed_roles:_R_POOL_OP
       ()
 
+  let deconfigure_repository_proxy = call
+      ~name:"deconfigure_repository_proxy"
+      ~in_product_since:rel_next
+      ~doc:"Clear the configured proxy for RPM package repositories."
```

Comment:
"Clear" is not the same as "deconfigure" - it reads like a contradiction. Clearing a cache means to reset it to an empty state whereas deconfigure means to remove it (in my understanding).